web/themes/custom/bglobal/source/_patterns/01-molecules/hero/_hero.scss

```diff
@@ -0,0 +1,74 @@
+.hero__title.h2 {
+  font-size: 3rem;
+  line-height: 5rem;
+}
+
+.hero__subtitle {
+  line-height: 1.5;
+}
+
+.hero__container {
+  background-color: #EEEEEE;
```

Comment:
Background color might not be necessary here,  check and remove if it is not needed.

Reply:
whoops, that was for testing. good catch.